ethcore/src/engines/vote_collector.rs

```diff
@@ -319,9 +338,9 @@ mod tests {
 		let collector = VoteCollector::default();
 		let round = 3;
 		// Vote is inserted fine.
-		assert!(full_vote(&collector, H520::random(), round, Some("0".sha3()), &Address::default()).is_none());
+		assert!(full_vote(&collector, H520::random(), round, Some("0".sha3()), &Address::default()));
 		// Returns the double voting address.
-		full_vote(&collector, H520::random(), round, Some("1".sha3()), &Address::default()).unwrap();
+		assert!(!full_vote(&collector, H520::random(), round, Some("1".sha3()), &Address::default()));
```

Comment:
this is what a double vote is: voting for two different hashes in the same voting round.